src/mpid/ch4/src/ch4_win.h

```diff
@@ -13,6 +13,30 @@
 
 #include "ch4_impl.h"
 
+static inline void MPIDI_create_workqs(MPIR_Win* win){
```

Comment:
It's a static function. I would even completely remove namespacing, but not the right time for this :)